django_mongodb/lookups.py

```diff
@@ -48,6 +50,14 @@ def in_(self, compiler, connection):
 def is_null(self, compiler, connection):
     if not isinstance(self.rhs, bool):
         raise ValueError("The QuerySet value for an isnull lookup must be True or False.")
+    if isinstance(self.lhs, Value):
```

Comment:
How did this come up in the context of this PR? What test does it fix?

Reply:
Should be in a separate PR, I forgot to move the code. It fixed a test but in a side effect way.